src/sage/interfaces/gap.py

```diff
@@ -1767,6 +1767,7 @@ def intmod_gap_to_sage(x):
 
 #############
 
+
 gap = Gap()
```

Comment:
We may remove the comment at L1768 and the other one above the class definition.